action.yaml

```diff
@@ -43,10 +43,6 @@ inputs:
     description: 'Namespace where the workflow template lies'
     required: false
     default: 'default'
-  delay-destroy:
-    description: 'Delay destruction of the setup by 1h'
-    required: True
-    default: 'No'
```

Comment:
If this is bool value, can we please use _True_ or _False_ here?

Reply:
@mbruzda-splunk, @kdoroszko-splunk  just  udated the snippet here and test the pipeline. its working without run_delay parameter